pages/verify.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Center, Text } from "@chakra-ui/react";
+import useLocalStorage from "@utils/useLocalStorage";
+
+export default function Verify(): JSX.Element {
```

Comment:
@feng-eric @victoranyun is the goal of this PR to implement the verify page to match the hi-fi design, or to just get something in ASAP for Sunday? If the former, I think there are some minor style changes that might need to be made to match the figma https://www.figma.com/file/ad7RNFErTGP7HkL52edTwQ/SDC-External-Application?node-id=1172%3A15150

Reply:
@feng-eric I'm not sure about this, please let me know

Reply:
Did most of it, just missing the back button now I believe. @jason17h, had a question regarding CSS and the footer btw. PM me when you see this